converter_ibkr.py

```diff
@@ -0,0 +1,391 @@
+import argparse
+import csv
+import os
+
+import pandas as pd
+from babel.numbers import parse_decimal
+
+from datetime import date
+
+
+parser = argparse.ArgumentParser(
+    description="Convert Interactive Brokers and Schwab CSV output to XLSX for later processing"
+)
+parser.add_argument(
```

Comment:
I'd create a separate converter.py file which only handles the instantiation of these IBKR/Schwab converters and pass the processed arguments list to them.

This way the user doesn't have to keep in mind which way to call which python file because it'd be enough to remember only one. It's easier for the README, too.

Also, I'm planning to add a Degiro converter (still planning, so not 100% sure) which would complicate things even further with the copy-pasted argument handling.

Reply:
Would be great if you could do that, yes. :) Wanted to simplify this a bit as this PR will touch a lot of things anyways.

Reply:
Okok, I'll do it in a follow-up PR